packages/datatrak-web/src/features/Survey/SurveyContext/utils.ts

```diff
@@ -199,22 +204,36 @@ const updateDependentQuestions = (
       }
     }
     if (hasArithmeticConfig(question)) {
-      const { config, questionId } = question;
-      const result = getArithmeticResult(expressionParser, formDataCopy, config.arithmetic);
+      const { arithmetic } = config as {
+        arithmetic: ArithmeticQuestionConfig;
+      };
+      const result = getArithmeticResult(expressionParser, formDataCopy, arithmetic);
```

Comment:
```suggestion
      const { arithmetic } = question.config;
      const result = getArithmeticResult(expressionParser, formDataCopy, arithmetic);
```
I'm considering banning the typecast operator :stuck_out_tongue_winking_eye: 

Jokes jokes, but I do prefer if we try avoid using it where possible. I find it makes the code a bit harder to parse, and it can introduce assumptions into the code. I prefer using [type guards](https://blog.logrocket.com/how-to-use-type-guards-typescript/) where needed as they consolidate the validation and typecasting into a single place, and make it clearer to read as well.

In this function, we get type safety from the `hasXConfig` functions. So best if we can make use of those :+1: